system/doc/design_principles/sup_princ.md

```diff
@@ -715,18 +706,19 @@ short as possible.
 
 ## Stopping
 
-Since the supervisor is part of a supervision tree, it is automatically
-terminated by its supervisor. When asked to shut down, it terminates all child
-processes in reversed start order according to the respective shutdown
-specifications, and then terminates itself.
+Since the supervisor is part of a supervision tree, it is
+automatically terminated by its supervisor. When asked to shut down, a
+supervisor terminates all child processes in reverse start order
+according to the respective shutdown specifications before terminating
```

Comment:
What does "according to the respective shutdown specifications" refer to? The shutdown timeouts?

Reply:
I suppose it refers to the `auto_shutdown` flag mentioned earlier.

Reply:
I will have to think about this. "Shutdown specifications" are mentioned under "Automatic Shutdown" without being formally defined.

Reply:
> I suppose it refers to the auto_shutdown flag mentioned earlier.

Hm, no, it can't be that... the `auto_shutdown` supervisor flag, in conjunction with the child spec flag `significant`, defines under what circumstances a supervisor shuts down automatically, by itself. But this paragraph is about supervisors which have been _asked_ to shut down.

> "Shutdown specifications" are mentioned under "Automatic Shutdown" without being formally defined.

Uh... then this was probably my doing, the auto shutdown feature and related documentation came with [EEP-56](https://www.erlang.org/eeps/eep-0056) :sweat_smile: